src/ResourceManager/Insights/Commands.Insights.Test/ScenarioTests/ActivityLogsTests.ps1

```diff
@@ -120,7 +120,71 @@ function Test-GetAzureSubscriptionIdLog
         $actual = Get-AzureRmLog -starttime 2015-01-15T12:30:00Z -endtime 2015-01-15T20:30:00Z 
 
         # Assert
-        Assert-AreEqual $actual.Count 1
+        Assert-AreEqual 1 $actual.Count
+    }
+    finally
+    {
+        # Cleanup
+        # No cleanup needed for now
+    }
+}
+
+<#
+.SYNOPSIS
+Tests getting the logs for a subscription Id.
+#>
+function Test-GetAzureSubscriptionIdLogMaxEvents
```

Comment:
Yes, that is one of the places where this is tested.
This is another scenario test for that. NOTE: the page here is set to 6 records, but the are 8 records in total.
**Test-GetAzureSubscriptionIdLogPaged** (same file as Test-GetAzureSubscriptionIdLogMaxEvents)

This is another place, the unit test below. Here the first page is 10 records and the second 5.
**GetAzureSubscriptionIdLogCommandParametersProcessing()**

I added more comments to clarify the goal of the individual tests and also added more cases to the existing scenario tests.